compute_tools/src/compute.rs

```diff
@@ -964,14 +966,14 @@ impl ComputeNode {
                     match &db {
                         DB::SystemDB => {}
                         DB::UserDB(db) => {
-                            url.set_path(db.name.as_str());
+                            conf.dbname(db.name.as_str());
                         }
                     }
 
-                    let url = Arc::new(url);
+                    let conf = Arc::new(conf);
```

Comment:
This code with a bunch of Arcs and clones looked a bit awkward to me even before my changes, but I haven't found a good way to change it quickly